src/main/resources/default/assets/tycho/styles/page.scss

```diff
@@ -106,7 +106,7 @@ nav.navbar {
   background-color: #fff;
 }
 
-@media (min-width: 700px) {
+@media (min-width: 750px) {
```

Comment:
Why don't we choose the same breakpoint as BS4, as in ≥768px? https://getbootstrap.com/docs/4.6/layout/overview/

Reply:
sounds good +1